### PR TITLE
Adding Mean Actor Critic

### LINKED ISSUE #11: Mean Actor-Critic
https://arxiv.org/pdf/1709.00503.pdf

### PR DESCRIPTION
**Pull request by RajGhugare19**

### CODE REVIEW

File: src/algorithms/policy_gradient/MAC.jl [hunk 1]
```diff
@@ -0,0 +1,132 @@
+export MACLearner
+
+using Flux
+
+"""
+    MACLearner(;kwargs...)
+   Keyword arguments
+- `approximator`::[`ActorCritic`](@ref)
+- `γ::Float32`, reward discount rate.
```
Comment (findmyway) [inline src/algorithms/policy_gradient/MAC.jl]: I think `bootstrap` is also configurable?

File: src/algorithms/policy_gradient/MAC.jl [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+Base.@kwdef mutable struct MACLearner{A<:ActorCritic} <:AbstractLearner
+    approximator::A
+    γ::Float32
+    max_grad_norm::Union{Nothing,Float32} = nothing
+    norm::Float32 = 0.0f0
+    actor_loss::Float32 = 0.0f0
+    critic_loss::Float32 = 0.0f0
+    loss::Float32 = 0.0f0
+    bootstrap::Bool = true          #if false then Q function is approximated using monte carlo returns
```
Comment (findmyway) [inline src/algorithms/policy_gradient/MAC.jl]: Move the comment into the doc string part.

**Review by findmyway — APPROVED** [review]
LGTM

**Comment by findmyway** [comment]
Well done! The README is also updated with this experiment added.

**Comment by RajGhugare19** [comment]
Thank you!!

**Pull request merged**